boards/riscv/longan_nano/Kconfig.defconfig

```diff
@@ -8,4 +8,14 @@ config BOARD
 	default "longan_nano" if BOARD_LONGAN_NANO
 	default "longan_nano_lite" if BOARD_LONGAN_NANO_LITE
 
+config DISK_DRIVER_SDMMC
+	bool
+	default y if DISK_DRIVERS
+
+# This definition automatically resolves the dependency to GPIO
+# from spi1/cs-gpios defined in the DTS file.
+config GPIO
+	bool
+	default y if $(dt_nodelabel_enabled,spi1) && $(dt_nodelabel_has_prop,spi1,cs-gpios)
```

Comment:
```suggestion
```

please just add `CONFIG_GPIO=y` to the board defconfig since GPIO is considered a _basic_ peripheral that can be enabled by default.

Reply:
Applied this pointing out.